### ISSUE TITLE
fix: Warning "Cannot stringify a function..." when using useFetch

### ISSUE THREAD

**Issue by sseeland** [nuxt-community/composition-api#133]
**🐛 The bug**
When I use `useFetch` inside of a `setup` function that returns other functions (e.g. as event handlers), when the page is rendered on the server, the server starts to throw warnings along the lines of "WARN  Cannot stringify a function bound onClick" (function name changes of course). This happens regardless of what I do in `useFetch` or if the exported function is actually used anywhere.

Interestingly, if I add a call to `console.log` anywhere in the `setup` function, the warning goes away, but the log is then printed twice (which doesn't happen without the `useFetch`).

**🛠️ To reproduce**
Minimal example that should reliably throw the warning on every page reload:

```
<template>
  <div />
</template>

<script lang="ts">
import { defineComponent, useFetch } from "nuxt-composition-api";

export default defineComponent({
  setup() {
    useFetch(() => {});

    function test() {}
    
    // uncomment this to silence the warning (and get 2 logs) 
    // console.log("test");

    return { test };
  },
});
</script>
```

**🌈 Expected behaviour**
Since this is the official approach to make event handlers available to the template and it's the way it's taught in the docs, this should just work without warnings.

**ℹ️ Additional context**
I noticed this first on the lastes release (0.10.2) but I can't say for sure if it happened before.

**Comment by danielroe** [comment]
@sseeland This should be fixed in 0.10.4. Let me know if you experience any other problems.

**Comment by sseeland** [comment]
Works for me as far as I can tell. Thanks for fixing!

The only other issue I'm running into right now is #19, which is already documented and has a PR, so all good.

Thanks for the great work!
